### DISCUSSION TITLE
Availability of release 12.9.1 on winget

### DISCUSSION THREAD

**Discussion by thetrotfreak** [JanDeDobbeleer/oh-my-posh#3025] [Troubleshoot] [answered]
Why the **oh-my-posh release** on *winget* is *stuck at 12.8.0*?

I would like to update to the latest release...

**Comment by JanDeDobbeleer** [comment, marked as answer]
@thetrotfreak careful, winget will display two sources. One is the store version, the other is the actual wibget version which should be up to date. The store is always a bit behind due to how it works for validation. Make sure to install using the following:

```
winget install JanDeDobbeleer.OhMyPosh -s winget
```

Both installers are identical btw, you won't have any conflicts.

**Comment by thetrotfreak** [comment]
@JanDeDobbeleer, thanks! Got the latest update `(12.9.1)`.
